review: Café au Lait

eliflores opened the pull request:
Solves #42

eliflores committed:
review: Café au Lait

Solves #42

eliflores commented:
Hi @lucasmdrs 
The review for Café au Lait is ready 🥞 
Happy to hear/read your comments ✨

lucasmdrs commented on content/en/reviews/cafe-au-lait.md:
```diff
@@ -0,0 +1,30 @@
+## The Review
+
+* The Pancakes 🥞
+    * Fluffiness: ⭐⭐⭐ - Not very fluffy.
```
I'm not sure I would give 3 :star: (out of 4) for my pancake fluffiness, although Fernando's pancake was fluffier :thinking: 

what do you think if we keep the score, but mention that there might be a variation on the fluffiness depending on the type of pancake you order?

eliflores replied:
Good point – I like the suggestion. I'll update the review ❤️

eliflores committed:
review: update the review of fluffiness

Add a note about the fluffiness that could be different depending on the pancakes that one orders.
